nodeenv.py

```diff
@@ -1052,8 +1060,10 @@ def main():
     if src_base_url is None:
         src_base_url = 'https://%s/download/release' % src_domain
 
-    if not opt.node or opt.node.lower() == "latest":
+    if opt.node.lower() == 'latest':
```

Comment:
to preserve the behaviour, this should be `if not opt.node or opt.node.lower() == 'latest'` -- I think the only case this matters is with `-n ''`

Reply:
Good point, done